### ISSUE TITLE
Unable to install due to an old version of symfony/console

### ISSUE THREAD

**Issue by EvgenyOrekhov** [ovr/phpsa#350]
When I try to install PHPSA with `composer require --dev ovr/phpsa:dev-master` I get the following output:

```text
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for ovr/phpsa dev-master -> satisfiable by ovr/phpsa[dev-master].
    - Conclusion: remove symfony/console v4.0.4
    - Conclusion: don't install symfony/console v4.0.4
    - ovr/phpsa dev-master requires symfony/console ^3.3.6 -> satisfiable by symfony/console[v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4].
    - Can only install one of: symfony/console[v3.3.10, v4.0.4].
    - Can only install one of: symfony/console[v3.3.11, v4.0.4].
    - Can only install one of: symfony/console[v3.3.12, v4.0.4].
    - Can only install one of: symfony/console[v3.3.13, v4.0.4].
    - Can only install one of: symfony/console[v3.3.14, v4.0.4].
    - Can only install one of: symfony/console[v3.3.15, v4.0.4].
    - Can only install one of: symfony/console[v3.3.16, v4.0.4].
    - Can only install one of: symfony/console[v3.3.6, v4.0.4].
    - Can only install one of: symfony/console[v3.3.7, v4.0.4].
    - Can only install one of: symfony/console[v3.3.8, v4.0.4].
    - Can only install one of: symfony/console[v3.3.9, v4.0.4].
    - Can only install one of: symfony/console[v3.4.0, v4.0.4].
    - Can only install one of: symfony/console[v3.4.1, v4.0.4].
    - Can only install one of: symfony/console[v3.4.2, v4.0.4].
    - Can only install one of: symfony/console[v3.4.3, v4.0.4].
    - Can only install one of: symfony/console[v3.4.4, v4.0.4].
    - Installation request for symfony/console (locked at v4.0.4) -> satisfiable by symfony/console[v4.0.4].
```

symfony/console should be updated to v4.0.4.

**Comment by EvgenyOrekhov** [comment]
I rewieved https://github.com/symfony/console/blob/master/CHANGELOG.md#400 and I didn't find any usages of the removed features in PHPSA.

And as with symfony/config, symfony/console doesn't support PHP 7.0: https://github.com/symfony/console/blob/4de97e8d0ff6d4822f0b4d1486b63edf9028eac8/composer.json#L19

I think the same change as in https://github.com/ovr/phpsa/commit/15cae55a4b84105ee252f847225f54404931a866 should be applied here as well.

**Comment by EvgenyOrekhov** [comment]
This issue should be closed if #352 is merged.
